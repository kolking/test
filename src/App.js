import React from 'react';
import { Provider } from 'react-redux';

import store from './store';

import FooListContainer from './containers/FooList';

export default function App() {
  return (
    <Provider store={store}>
      <FooListContainer />
    </Provider>
  );
}
