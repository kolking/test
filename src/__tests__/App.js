import React from 'react';
import TestRenderer from 'react-test-renderer';

/**
 * Under test
 */
import App from '../App';

const createElement = props => <App {...props} />;
const createRenderer = props => TestRenderer.create(createElement(props));

it('should render normally', done => {
  const tree = createRenderer({});

  setImmediate(() => {
    expect(tree).toMatchSnapshot();

    done();
  });
});
