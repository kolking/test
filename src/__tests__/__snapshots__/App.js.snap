// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render normally 1`] = `
<section
  className="container"
>
  <hgroup
    className="container_header"
  >
    <h1>
      List of foos
    </h1>
    <button
      onClick={[Function]}
      type="button"
    >
      Create New
    </button>
  </hgroup>
  <div
    className="container_content"
  >
    <p
      className="container_empty"
    >
      There are no foos
    </p>
  </div>
</section>
`;
